emulsify.breakpoints.yml

```diff
@@ -1,16 +1,16 @@
 # Breakpoints - change based on site need
```

Comment:
@evanmwillhite Why did you reverse the order of these? I thought they had to be largest to smallest in order for things to work correctly...

Reply:
my experience was the opposite. It didn't work until I did this. Can you confirm in your testing?